forestserviceprototype/specialuseform/management/commands/create_permits.py

```diff
@@ -0,0 +1,20 @@
+#! /usr/bin/env python3
+from django.core.management.base import BaseCommand, CommandError
+from specialuseform.factories import PermitFactory
+
+
+class Command(BaseCommand):
```

Comment:
Done in https://github.com/18F/forest-service-prototype/pull/57/commits/2d451051b89d92962f4442f86b0498a134c5ec55.